conans/test/functional/old/conan_trace_file_test.py

```diff
@@ -123,7 +123,7 @@ def test_trace_actions(self):
             self.assertEqual(num_post, 2)  # 2 get urls
 
         num_get = len([it for it in actions if "REST_API_CALL" in it and "GET" in it])
-        self.assertEqual(num_get, 10)
+        self.assertEqual(num_get, 9)
```

Comment:
We save a "ping" (counted only when the call doesn't raise 401), because we fail sooner to authenticate:

0 = {str} 'http://fake2d36cd11d8974824a04482cf91991095.com/v1/users/authenticate'
1 = {str} 'http://fake2d36cd11d8974824a04482cf91991095.com/v1/ping'
2 = {str} 'http://fake2d36cd11d8974824a04482cf91991095.com/v1/conans/Hello0/0.1/lasote/stable/packages/d3f1f0cd90ebac9640b57627a960c07e557d3cca/download_urls'
3 = {str} 'http://fake2d36cd11d8974824a04482cf91991095.com/v1/ping'
4 = {str} 'http://fake2d36cd11d8974824a04482cf91991095.com/v1/conans/Hello0/0.1/lasote/stable/digest'
5 = {str} 'http://fake2d36cd11d8974824a04482cf91991095.com/v1/users/check_credentials'
6 = {str} 'http://fake2d36cd11d8974824a04482cf91991095.com/v1/conans/Hello0/0.1/lasote/stable'
7 = {str} 'http://fake2d36cd11d8974824a04482cf91991095.com/v1/users/check_credentials'
8 = {str} 'http://fake2d36cd11d8974824a04482cf91991095.com/v1/conans/Hello0/0.1/lasote/stable/packages/d3f1f0cd90ebac9640b57627a960c07e557d3cca'



00 = {str} 'http://fake1fe511ccdec1410a98066be81b858270.com/v1/ping'
01 = {str} 'http://fake1fe511ccdec1410a98066be81b858270.com/v1/users/authenticate'
02 = {str} 'http://fake1fe511ccdec1410a98066be81b858270.com/v1/ping'
03 = {str} 'http://fake1fe511ccdec1410a98066be81b858270.com/v1/conans/Hello0/0.1/lasote/stable/packages/d3f1f0cd90ebac9640b57627a960c07e557d3cca/download_urls'
04 = {str} 'http://fake1fe511ccdec1410a98066be81b858270.com/v1/ping'
05 = {str} 'http://fake1fe511ccdec1410a98066be81b858270.com/v1/conans/Hello0/0.1/lasote/stable/digest'
06 = {str} 'http://fake1fe511ccdec1410a98066be81b858270.com/v1/users/check_credentials'
07 = {str} 'http://fake1fe511ccdec1410a98066be81b858270.com/v1/conans/Hello0/0.1/lasote/stable'
08 = {str} 'http://fake1fe511ccdec1410a98066be81b858270.com/v1/users/check_credentials'
09 = {str} 'http://fake1fe511ccdec1410a98066be81b858270.com/v1/conans/Hello0/0.1/lasote/stable/packages/d3f1f0cd90ebac9640b57627a960c07e557d3cca'